src/main/webapp/user-page.html

```diff
@@ -16,194 +16,304 @@
+      }
+
+      function reBuildUsername() {
+        const usernameContainers = document.querySelectorAll(`[id='username']`);
+        usernameContainers.forEach(div => {
+          $.getScript("/js/message-loader.js", () => {
+            div.innerHTML = getUsername(
+              new URLSearchParams(window.location.search).get("user")
+            );
+          });
         });
-    }
-
-    function reBuildUsername() {
-      const usernameContainers = document.querySelectorAll(`[id='username']`);
-      usernameContainers.forEach(div => {
-        $.getScript("/js/message-loader.js", () => {
-          div.innerHTML = getUsername(
-            new URLSearchParams(window.location.search).get("user")
-          );
+      }
+
+      function displayUploadedImage(input) {
+        if (input.files && input.files[0]) {
+          var reader = new FileReader();
+
+          reader.onload = function(e) {
+            document
+              .getElementById("uploaded-image-container")
+              .classList.remove("hidden");
+            $("#uploaded-image").attr("src", e.target.result);
+          };
+
+          reader.readAsDataURL(input.files[0]);
+        }
+      }
+
+      function initMap() {
+        var input = document.getElementById("searchMapInput");
+        var autocomplete = new google.maps.places.Autocomplete(input);
+        autocomplete.addListener("place_changed", function() {
+          var place = autocomplete.getPlace();
         });
-      });
-    }
-
-    function initMap() {
-      var input = document.getElementById("searchMapInput");
-      var autocomplete = new google.maps.places.Autocomplete(input);
-      autocomplete.addListener("place_changed", function () {
-        var place = autocomplete.getPlace();
-      });
-    }
-  </script>
-</head>
-
-<body onload="buildUI();">
-  <div class="content-container mt-7 pt-4">
-    <div class="left-container pr-4 position-fixed">
-      <form id="profile-image-form" method="POST" class="mb-5" enctype="multipart/form-data">
-        <div class="profile-image-container">
-          <img id="user-profile-image" alt="User Profile Image" class="rounded border" />
-          <label class="edit-profile-image p-1 m-0 border-0 btn btn-default btn-file hidden" id="edit-profile-image">
-            <i class="fas fa-pencil-alt fa-2x"></i>
-            <input type="file" name="image" accept="image/*" id="profile-image-upload-input" style="display: none;" />
-          </label>
-        </div>
-      </form>
+      }
+    </script>
+  </head>
 
-      <div class="profile-container mt-5" id="profile-container">
-        <div id="profile-info-container">
-          <div class="border-bottom pb-3 pl-1">
-            <h4 class="card-title" id="username"></h4>
-            <h6 class="font-weight-lighter text-muted" id="aboutme"></h6>
+  <body onload="buildUI();">
+    <div class="content-container mt-7 pt-4">
+      <div class="left-container pr-4 position-fixed">
+        <form
```

Comment:
Hey, can you reformat this HTML page

Reply:
Sure! I think there were something wrong with my plugin Prettifier's settings.

Reply:
> Hey, can you reformat this HTML page

Hi, I have reformated it. Can help approve it? Thanks!